setup.php

```diff
@@ -95,6 +95,8 @@ function plugin_init_order() {
    if ($plugin->isActivated('order')) {
       $PLUGIN_HOOKS['migratetypes']['order'] = 'plugin_order_migratetypes';
 
+      $PLUGIN_HOOKS['pre_show_item']['order'] = ['PluginOrderOrder', 'preShowItem'];
```

Comment:
Why don't you just add a check on PluginOrderOrder::showForm() directly ? By doing this check directly there, you would be able to prevent displaying of the form when configuration is not set.

Reply:
agree with that

![image](https://user-images.githubusercontent.com/7335054/121658686-de1e3880-caa1-11eb-8192-a4632106b35d.png)